pkg/ebpf/tracer.go

```diff
@@ -116,11 +116,25 @@ func NewTracer(config *Config) (*Tracer, error) {
 	}
 	log.Infof("socket struct offset guessing complete (took %v)", time.Since(start))
 
+	// check if current platform is Rhel or CentOS because it affects what kprobe are we going to enable
+	isRhel, err := isRhelOrCentOS()
+	if err != nil {
+		return nil, err
```

Comment:
I don't think we should fail hard here - in what cases can this throw an error?

Reply:
I think it could be more explicit, like doing an `errors.Wrap` with more info, but essentially the reason this returns an error is that since we cannot reliably determine which platform current host is, we cannot pick the right kprobes to attach(regular tcp_sendmsg vs. RHEL version of tcp_sendmsg). How does that sound?

Reply:
I don't think that's enough, we've already seen issues in the past for retrieving the platform. Failing hard here sounds like it'll cause more issues than not.

Lets log something and default to the regular `tcp_sendmsg` if we're unable to detect the platform.

Reply:
Actually, it might be better to log the case where we actually detect RHEL instead of when we can't detect the platform (which already has it's own log output I believe). 

`Detected RHEL/CentOS, switching to 3.3.x-specific kprobes...`

Reply:
I see, that's good to know. I can make it work like that.